glommio/src/read_write_lock.rs

```diff
@@ -436,10 +450,10 @@ impl<T> ReadWriteLock<T> {
         Ok(unsafe { &mut *(state.value.get()) })
     }
 
-    /// Locks this ReadWriteLock with shared read access, blocking the current fiber
-    /// until it can be acquired.
+    /// Locks this ReadWriteLock with shared read access, suspending the current fiber
+    /// until lock can be acquired.
```

Comment:
good! I didn't comment on that, but a pet peeve of mine just so you know:
I think the words "it" and "this" should be avoided whenever possible. Most of the time they are ambiguous and it is not clear to which noun they refer to.
(your usage of "this" above is clear enough because it is attached to `ReadWriteLock`)

Reply:
Completely agree, seem I missed that. I do not like it too. Sometimes use automatically though. It is not clear what "it" meanst. I will be a bit of busy at the bigging of next week will provide the next PR in the middle of next week.